### PR TITLE
Chore: ggircs DAGs reside in ggircs repository

### PR DESCRIPTION
**Pull request by pbastia**
Moving the ggircs DAGs to the ggircs repository, and using the airflow dag trigger chart to keep them up-to-date on deploy.

### CODE REVIEW

File: helm/cas-ggircs/Chart.yaml [hunk 1]
```diff
@@ -8,3 +8,16 @@ dependencies:
...
+    alias: download-ggircs-dags
+  - name: cas-airflow-dag-trigger
+    version: 1.0.6
+    repository: https://bcgov.github.io/cas-airflow
+    alias: download-swrs-dags
+    condition: download-swrs-dags.enabled
+  - name: cas-airflow-dag-trigger
+    version: 1.0.6
+    repository: https://bcgov.github.io/cas-airflow
+    alias: trigger-ggircs-deploy-db
```
Comment (matthieu-foucault) [inline helm/cas-ggircs/Chart.yaml:23]: I think you have to run a `helm dep up` to update the `Chart.lock` file

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]
one small change

File: helm/cas-ggircs/Chart.yaml (hunk 1 above)
Comment (pbastia) [reply helm/cas-ggircs/Chart.yaml:23]: 👍 done!

**Review by matthieu-foucault — DISMISSED** [review]
@pbastia just needs a rebase

File: dags/cas_ggircs_swrs_dags.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        python_callable=trigger_k8s_cronjob,
+        task_id='ggircs_read_only_user',
+        op_args=['cas-ggircs-db-create-readonly-user', namespace],
+        dag=dag)
+
+
+def trigger_ciip_deploy_db_dag(dag):
+    return TriggerDagRunOperator(
+        task_id='trigger_ciip_deploy_db_dag',
+        trigger_dag_id="ciip_deploy_db",
```
Comment (naomiaro) [inline dags/cas_ggircs_swrs_dags.py:80]: this dag will/is being renamed to cas_ciip_portal_deploy_db

Comment (naomiaro) [reply dags/cas_ggircs_swrs_dags.py:80]: https://cas-airflow-dev.apps.silver.devops.gov.bc.ca/graph?dag_id=cas_ciip_portal_deploy_db

File: dags/cas_ggircs_swrs_dags.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    'start_date': START_DATE
+}
+
+DAG_ID = "cas_ggircs_swrs_eccc"
+SCHEDULE_INTERVAL = '0 8 * * *'
+
+dag_incremental = DAG(DAG_ID + '_incremental', schedule_interval=SCHEDULE_INTERVAL,
+                      default_args=default_args, user_defined_macros={'json': json}, start_date=START_DATE)
+dag_full = DAG(DAG_ID+'_full', schedule_interval=None,
+               default_args=default_args)
```
Comment (matthieu-foucault) [inline dags/cas_ggircs_swrs_dags.py:38]: @naomiaro  let's add `is_paused_upon_creation=True` to these two DAGs

**Comment by naomiaro** [comment]
The flag `--wait-for-jobs` is new in Helm 3.5.0 https://github.com/roboll/helmfile/issues/1650, so I updated the circleCI version here. Might just need to check that we have a new enough helm version installed where needed potentially.

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**